### PR TITLE
Add table listing the predefined Jenkins job

### PR DESCRIPTION
**Pull request by sbesson**
This should start providing minimal documentation on the devspace jobs, their
context and their tasks.

### CODE REVIEW

**Comment by aleksandra-tarkowska** [comment]
as this was tested with compose 1.6.2 only, worth adding version requirements? Compose 1.6.2 requires Docker Engine 1.9.1 or later.

**Comment by sbesson** [comment]
Pushed

**Comment by aleksandra-tarkowska** [comment]
looks good, good to merge

File: README.md [hunk 1]
```diff
@@ -38,3 +46,21 @@ Getting started
  *  Configure artifactory:
     - Add an artifactory user (optional)
     - Under "System Configuration" add your artifactory URL
+
+## Job workflow
+
+The default deployment initializes a Jenkins server with a [predefined set of
+jobs](homes/jobs). The table below lists the job names, the Jenkins node labels they are associated to and a short description of their:
```
Comment (aleksandra-tarkowska) [inline README.md]: that seems to be too long, I can fix it in my next PR

**Comment by aleksandra-tarkowska** [comment]
looks good, good to merge

**Pull request merged**